public/static/app/perf/perf.js

```diff
@@ -329,7 +329,10 @@ $http.get(templateUrl).success(function(template) {
   }
 
   $scope.comparisonPct = function (compareSample, testName) {
-    const threadLevel = _.max($scope.selectedThreads[testName]);
+    const maxThreadLevel = "MAX";
+    let threadLevel = _.contains($scope.selectedThreads[testName], maxThreadLevel) ?
+      _.max($scope.hoverSamples[testName].threadResults, (result) => result.threadLevel).threadLevel :
+      _.max($scope.selectedThreads[testName]);
```

Comment:
[nit] can you put a comment about what this does, it's not clear.

Note: not relevant to the current PR but ... see EVG-3009, the current implmentation is wrong. Max thread level shouldn't be the highest thread level, it should have been the highest result (this should be read from the database as it is available).

Reply:
added a comment

Reply:
:+1: thanks.